variables.tf

```diff
@@ -31,6 +31,12 @@ variable "hd" {
   default     = null
 }
 
+variable "base_url" {
+  type        = string
+  description = "The base URL of your auth provider. Applicable to Auth0 and Okta. This must begin with http(s)."
+  default     = null
```

Comment:
If this isn't set, Terraform can't interpolate the null value into a string template.